packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -17,23 +17,32 @@ import {
   getGreyShade,
 } from '../globalStyles';
 import { EntityItem, ITEM_HEIGHT } from './EntityItem';
+import { fetchEntities } from './helpers';
 
 const SEARCH_BOX_HEIGHT = 40;
+const INITIAL_NUMBER_TO_SHOW = 1000; // will load more if they scroll to the bottom of the list
+const LOADING_MESSAGE_SECTION = { title: 'Loading more results...', data: [] }; // slightly hacky way of showing a loading message at the bottom of the list until more lazily load
+const NO_RESULTS_SECTION = { title: 'No results', data: [] };
+
 export class EntityList extends PureComponent {
   constructor(props) {
     super(props);
+    this.baseQuery = fetchEntities(
```

Comment:
I feel `baseQuery` is maybe a tad vague? 
```suggestion
    this.baseEntitiesQuery = fetchEntities(
```